tobler/data.py

```diff
@@ -0,0 +1,46 @@
+import quilt3
+from urllib.parse import unquote
+
+def store_rasters():
+    """Save raster data to the local quilt package storage.
+
+    Returns
+    -------
+    None
+        Data will be available in the local quilt registry and available
+        for use in interpolation functions from the `harmonize` module.
+
+    """
+    quilt3.Package.install("rasters/nlcd", "s3://quilt-cgs")
+
+
+def fetch_quilt_path(path):
+    """utility for getting the path to a raster stored with quilt.
+
+    Parameters
+    ----------
+    path : str
```

Comment:
Shouldn't be better to inform here what are the possible options of the function (i.e. ["nlcd_2011", "nlcd_2001"])?

Reply:
good idea. this also reminds me there's a 2016 vintage i need to convert/upload to quilt

Reply:
i added a more thorough description. Are you ok with merging so i can test whether i configured https://github.com/spatialucr/geosnap/pull/126 properly?